### DISCUSSION TITLE
Integration with models hosted on TogetherAPI or other services.

### DISCUSSION THREAD

**Discussion by nuvoqueonline** [FlowiseAI/Flowise#929] [Q&A]
Hi, 

Is it possible to use Models hosted on a service like Together API (or any other hosted models) as a Language Model in the conversational retrieval QA chain. 

I am able to use models available on Replicate, however was wondering if there is a way to access models hosted on services other than Local/ Huggingface / Replicate.

**Comment by HenryHengZJ** [comment]
You will probably have to create another node similar to Replicate to use TogetherAPI
